src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -755,19 +759,22 @@ void Engine::AddExtension(const InferenceEngine::IExtensionPtr& extension) {
 QueryNetworkResult Engine::QueryNetwork(const CNNNetwork& network, const std::map<std::string, std::string>& config) const {
     WeightsSharing::Ptr fake_w_cache;
 
+    auto model = network.getFunction();
+    if (model == nullptr) {
+        IE_THROW() << "Only ngraph-based models are supported!";
+    }
+
     Config conf = engConfig;
-    conf.readProperties(config);
+    Config::NetworkType networkType = ov::op::util::has_op_with_type<ngraph::op::v1::Convolution>(model) ||
```

Comment:
I think it is worth wrap it into a function getNetworkType()

Reply:
moved it to `cpu_utils.hpp`